translate/src/modules/entitydetails/components/Metadata.tsx

```diff
@@ -258,19 +210,7 @@ export function Metadata({
         <ContextIssueButton openTeamComments={openTeamComments} />
       )}
       <Screenshots source={entity.comment} locale={code} />
-      <OriginalStringProxy
-        entity={entity}
-        terms={terms}
-        handleClickOnPlaceable={handleClickOnPlaceable}
-      />
-      {popupTerms.length > 0 && (
-        <TermsPopup
-          isReadOnlyEditor={isReadOnlyEditor}
-          terms={popupTerms}
-          hide={hidePopupTerms}
-          navigateToPath={navigateToPath}
-        />
-      )}
+      <OriginalString navigateToPath={navigateToPath} terms={terms} />
```

Comment:
`OriginalString` (as well as `ContextIssueButton` above) are not really string Metadata. We should rename this component to something that also covers these two components. That said, I don't have a great idea what would that name be.